metals-docs/src/main/scala/docs/Snapshot.scala

```diff
@@ -42,7 +42,8 @@ object Snapshot {
   private def current: Snapshot =
     Snapshot(BuildInfo.metalsVersion, LocalDateTime.now())
 
-  /** Returns the latest published snapshot release, or the current release if. */
+  /**
+   * Returns the latest published snapshot release, or the current release if. */
```

Comment:
I also don't love these, but since they aren't `//` they make sense.

Reply:
I think it's fine, but if you prefer otherwise, I don't really have a preference.

Reply:
I'd say if you don't have a preference, then let's just leave them as is 👍